scio-test/src/test/scala/com/spotify/scio/io/dynamic/DynamicFileTest.scala

```diff
@@ -98,7 +99,7 @@ class DynamicFileTest extends PipelineSpec {
   it should "support generic Avro files" in {
     val tmpDir = Files.createTempDirectory("dynamic-io-")
     val sc1 = ScioContext()
-    implicit val coder = Coder.avroGenericRecordCoder(schema)
+    implicit val coder: Coder[GenericRecord] = Coder.avroGenericRecordCoder(schema)
```

Comment:
undo?

Reply:
I prefer explicit types on implicit values, but I will revert that 😺 .